README.md

```diff
@@ -49,3 +49,41 @@ Create a kubernetes manifest for a pod which will containa ToDo app container:
 1. `README.md` Should have explained your strategy configuration (Why such numbers)
 1. `README.md` Should have explained how to access the app after deployment
 1. Create PR with your changes and attach it for validation on a platform.
+
+
+In order to deploy the app to k8s you should execute the next steps:
+Create namespace
+You can edit the file namespace.yml and change value 'todoapp' on 'mateapp' and then execute the next command
+kubectl apply -f namespace.yml
```

Comment:
This command will deploy todoapp namespace.

Reply:
I changed readme file and left one command for creating namespace
But if edit namespace.yaml file and will write their 'mate app' and then apply this file we also create the namespace